src/NServiceBus.RavenDB.Tests/Persistence/DocumentStoreExtensionsForVoron.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.RavenDB.Tests.Persistence
+{
+    using System;
+    using System.Collections.Generic;
+    using Raven.Abstractions.Data;
+    using Raven.Client.Document;
+
+    internal static class DocumentStoreExtensionsForVoron
+    {
+        public static IDisposable SetupVoronTest(this DocumentStore store)
+        {
+            store.Initialize();
+            store.DatabaseCommands.GlobalAdmin.CreateDatabase(new DatabaseDocument
+            {
+                Id = "VoronTest",
+                Settings = new Dictionary<string, string>
+                    {
+                        { "Raven/StorageTypeName", "voron" },
+                        { "Raven/DataDir", "~/VoronTest" }
```

Comment:
Yes there is a test Yak in need of your capable shaving :)

http://builds.particular.net/viewType.html?buildTypeId=ProductionStagingTests_RavenDBPersistenceCompatibility

On Tue, May 5, 2015 at 5:10 AM, Daniel Marbach <notifications@github.com>
wrote:

> In
> src/NServiceBus.RavenDB.Tests/Persistence/DocumentStoreExtensionsForVoron.cs
> <https://github.com/Particular/NServiceBus.RavenDB/pull/68#discussion_r29641670>
> :
>
> > +    using System.Collections.Generic;
> > +    using Raven.Abstractions.Data;
> > +    using Raven.Client.Document;
> > +
> > +    internal static class DocumentStoreExtensionsForVoron
> > +    {
> > +        public static IDisposable SetupVoronTest(this DocumentStore store)
> > +        {
> > +            store.Initialize();
> > +            store.DatabaseCommands.GlobalAdmin.CreateDatabase(new DatabaseDocument
> > +            {
> > +                Id = "VoronTest",
> > +                Settings = new Dictionary<string, string>
> > +                    {
> > +                        { "Raven/StorageTypeName", "voron" },
> > +                        { "Raven/DataDir", "~/VoronTest" }
>
> I'll change that. Unrelated question: Do I need to update wire compat
> tests or anything else as well somewhere before releasing? Am Montag, 4.
> Mai 2015 schrieb Simon Cropp :
> … <#14d220d60b8941f3_>
>  In
> src/NServiceBus.RavenDB.Tests/Persistence/DocumentStoreExtensionsForVoron.cs
> <
> https://github.com/Particular/NServiceBus.RavenDB/pull/68#discussion_r29625639>
> : > + using System.Collections.Generic; > + using Raven.Abstractions.Data;
> > + using Raven.Client.Document; > + > + internal static class
> DocumentStoreExtensionsForVoron > + { > + public static IDisposable
> SetupVoronTest(this DocumentStore store) > + { > + store.Initialize(); > +
> store.DatabaseCommands.GlobalAdmin.CreateDatabase(new DatabaseDocument > +
> { > + Id = "VoronTest", > + Settings = new Dictionary<string, string> > + {
> > + { "Raven/StorageTypeName", "voron" }, > + { "Raven/DataDir",
> "~/VoronTest" } Test should be able to run in parallel. Perhaps use a temp
> directory? — Reply to this email directly or view it on GitHub <
> https://github.com/Particular/NServiceBus.RavenDB/pull/68/files#r29625639>
> .
>  -- Daniel Marbach Solution Architect, Particular Software Email:
> daniel.marbach@particular.net Web: www.particular.net <
> http://t.signauxneuf.com/e1t/c/5/f18dQhb0S7lC8dDMPbW2n0x6l2B9nMJN7t5XX4RzDjlVS9QB88pTc_dW3Ljg7x56dWyBdgBXs402?t=http%3A%2F%2Fwww.particular.net%2F&si=4896787407044608&pi=33cc6202-1f31-4040-8d6c-e70641f44bba
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus.RavenDB/pull/68/files#r29641670>
> .
>